python/py-botocore/Portfile

```diff
@@ -19,11 +19,11 @@ long_description    A low-level interface to a growing number of Amazon Web \
 
 homepage            https://github.com/boto/botocore
 
-checksums           rmd160  8b8256bdf4a3f029745a68b991c7cb80a107976b \
-                    sha256  1909424c9544f92142c8e551888731e32a99f9c99cfe8d21fea3ec0c32981dae \
-                    size    6095336
+checksums           rmd160  efed8962d191566c52c49ef9f780dda2a8c3acd8 \
+                    sha256  3fb144d2b5d705127f394f7483737ece6fa79577ca7c493e4f42047ac8636200 \
+                    size    6706807
 
-python.versions     27 35 36 37 38
+python.versions     38
```

Comment:
This will break other ports that use botocore, such as py-awscli. If you want to pursue removing old python versions here, you need to do it in all dependencies first. Basically, don't remove a port that other ports still depend on.

Reply:
Ok I see you removed pre-3.8 python versions from py-awscli in #8113, and I merged it because no other port depends on py-awscli. But that was just the first one I found that depended on pre-py38 awscli. The next one is py-boto3, and there are others.

Reply:
Reversed